Source/WebCore/dom/BroadcastChannel.cpp

```diff
@@ -66,7 +66,7 @@ static HashMap<BroadcastChannelIdentifier, ScriptExecutionContextIdentifier>& ch
 static PartitionedSecurityOrigin partitionedSecurityOriginFromContext(ScriptExecutionContext& context)
 {
     Ref securityOrigin { *context.securityOrigin() };
-    Ref topOrigin { context.settingsValues().broadcastChannelOriginPartitioningEnabled ? context.topOrigin() : securityOrigin.get() };
+    Ref topOrigin { context.topOrigin() };
     return { WTFMove(topOrigin), WTFMove(securityOrigin) };
```

Comment:
```suggestion
    return { context.topOrigin(), context.securityOrigin().releaseNonNull() };
```
We might be able to do a future refactoring and maybe remove `partitionedSecurityOriginFromContext`, `PartitionedSecurityOrigin` and instead use `ClientOrigin` and `Document::clientOrigin()` instead.

I would expect the BroadcastChannel client origin to be the same as the client origin of the Document tied to the BroadcastChannel context.